From 45bb666b284ca906e2b8a96ec4dd6168643d6330 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?P=C3=A1draig=20Brady?= <P@draigBrady.com>
Date: Thu, 17 Jul 2014 11:59:16 +0100
Subject: [PATCH] remove runtime dep on python-pbr

---
 sahara/version.py | 15 +++++++++++++--
 1 file changed, 13 insertions(+), 2 deletions(-)

diff --git a/sahara/version.py b/sahara/version.py
index ba79883..fd70c87 100644
--- a/sahara/version.py
+++ b/sahara/version.py
@@ -13,6 +13,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from pbr import version
 
-version_info = version.VersionInfo('sahara')
+class VersionInfo(object):
+    release = 'REDHAT_SAHARA_RELEASE'
+    version = 'REDHAT_SAHARA_VERSION'
+    package = 'sahara'
+
+    def version_string(self):
+        return self.version
+
+    def release_string(self):
+        return self.release
+
+
+version_info = VersionInfo()
