--- sahara/version.py	2014-05-05 09:18:09.682115488 -0400
+++ sahara/version.py	2014-05-05 09:34:46.248635372 -0400
@@ -13,6 +13,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from pbr import version
 
-version_info = version.VersionInfo('sahara')
+class VersionInfo(object):
+    release = 'REDHAT_SAHARA_RELEASE'
+    version = 'REDHAT_SAHARA_VERSION'
+    package = 'sahara'
+
+    def version_string(self):
+        return self.version
+
+    def release_string(self):
+        return self.release
+
+
+version_info = VersionInfo()
